OsmAnd/res/values/strings.xml

```diff
@@ -34,14 +34,14 @@
     <string name="monitoring_min_speed">Minimum speed</string>
     <string name="monitoring_min_accuracy">Minimum accuracy</string>
     <string name="monitoring_min_distance">Minimum displacement</string>
-    <string name="tracks_view_path">Menu — My places — Tracks</string>
-    <string name="multimedia_notes_view_path">Menu — My places — Notes</string>
-    <string name="osm_edits_view_path">Menu — My places — OSM Edits</string>
-    <string name="reset_plugin_to_default">Reset plugin settings to default</string>
+    <string name="tracks_view_path">Menu → My places → Tracks</string>
+    <string name="multimedia_notes_view_path">Menu → My places → Notes</string>
+    <string name="osm_edits_view_path">Menu → My places → OSM edits</string>
+    <string name="reset_plugin_to_default">Reset plugin settings to defaults</string>
```

Comment:
I welcome implementing a convention how we state menu paths. Arrows one possible way. Maybe open an issue to find a consensus, and then we can implement in one change to harmonize overall?

Reply:
This is what was used previously in other strings, leaving it now consistent. If so they could all be changed and one would only have to search for the arrow.

Reply:
Apologies, I had lost track of that issue, and it looks like noone else commentrd there. Will review at some point and then we try to resolve.